etstool.py

```diff
@@ -254,6 +254,21 @@ def install(runtime, toolkit, environment, editable, source):
     click.echo('Done install')
 
 
+@cli.command()
+@click.option("--runtime", default="3.6", help="Python version to use")
+@click.option("--toolkit", default="pyqt", help="Toolkit and API to use")
+@click.option("--environment", default=None, help="EDM environment to use")
+def update(runtime, toolkit, environment):
```

Comment:
Now this `update` is defined twice... Flake8 would have caught this but this file is not currently being flaked.